### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by edward3sc**
Changes to satisfy support ticket [SP-1399](https://3sidedcube.atlassian.net/browse/SP-1399)

### CODE REVIEW

File: app/src/services/areas.service.js [hunk 1]
```diff
@@ -23,8 +23,18 @@ class AreasService {
...
+        // exclude areas that have wdpaid or admin object keys
+        const areasToReturn = areas.data.filter(area => {
+          let wdpa = false;
+          if (area.attributes.wdpaid) wdpa = true;
+          let gadm = false;
+          if (area.attributes.admin && Object.keys(area.attributes.admin).length > 0) gadm = true;
+          return !(wdpa || gadm);
+        });
+        return areasToReturn;
+      } else return [];
```
Comment (correcthorsebatterystaple) [inline app/src/services/areas.service.js]:
```suggestion
      }
      return [];
```

File: app/src/services/areas.service.js [hunk 2]
```diff
@@ -23,8 +23,18 @@ class AreasService {
...
-      logger.info("User areas", areas);
-      return areas && areas.data;
+      logger.info("got user areas", areas);
+      if (areas && areas.data) {
+        // exclude areas that have wdpaid or admin object keys
+        const areasToReturn = areas.data.filter(area => {
+          let wdpa = false;
+          if (area.attributes.wdpaid) wdpa = true;
+          let gadm = false;
+          if (area.attributes.admin && Object.keys(area.attributes.admin).length > 0) gadm = true;
```
Comment (correcthorsebatterystaple) [inline app/src/services/areas.service.js]:
```suggestion
          const wdpa = !!area.attributes.wdpaid;
          const gadm = !!area.attributes.admin && Object.keys(area.attributes.admin).length > 0;
```

File: app/src/services/areas.service.js [hunk 3]
```diff
@@ -23,8 +23,18 @@ class AreasService {
...
+          return !(wdpa || gadm);
```
Comment (correcthorsebatterystaple) [inline app/src/services/areas.service.js]:
```suggestion
          return !wdpa && !gadm;
```

File: docker-compose-develop.yml [hunk 4]
```diff
@@ -65,7 +65,7 @@ services:
       - gfw-network
 
   mongo:
-    image: mongo:3.4
+    image: mongo:3.6
```
Comment (correcthorsebatterystaple) [inline docker-compose-develop.yml:68]: Any breaking changed to worry about?

**Review by correcthorsebatterystaple — CHANGES_REQUESTED** [review]

File: docker-compose-develop.yml (hunk 4 above)
Comment (edward3sc) [reply docker-compose-develop.yml:68]: Nope

**Review by wxllJK — APPROVED** [review]

**Review by owenjs — APPROVED** [review]

**Review by correcthorsebatterystaple — APPROVED** [review]

**Pull request merged**